<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKANBrowserDialogSettings</class>
 <widget class="QDialog" name="CKANBrowserDialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dlg_set_title</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QgsCollapsibleGroupBox" name="IDC_grpBox_cache_folder">
     <property name="title">
      <string>dlg_settings_grp_cache_folder</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="IDC_pbHelpCache">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>dlg_set_tool_cache</string>
           </property>
           <property name="text">
            <string>?</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>dlg_set_cache_path</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="IDC_leCacheDir">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_bCacheDir">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>dlg_set_ellipsis</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="IDC_grpBox_authentication">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>dlg_settings_grp_authentication</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="IDC_lblAuthCfg">
        <property name="text">
         <string>Authentication</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IDC_leAuthCfg">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="IDC_bAuthCfgEdit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_bAuthCfgClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IDC_cbAuthPropagate">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Warning: &lt;/span&gt;if this option is enabled, the same authentication configuration will be implicitely propagated to &lt;span style=&quot; font-weight:600;&quot;&gt;all&lt;/span&gt; OGC layers (WMS, WFS ...) returned by this catalog, this may leak credentials to the result endpoint  if it is not supposed to use the same authentication configuration.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Use the same authentication settings for all OGC layers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
     <property name="title">
      <string>dlg_settings_grpbox_misc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="IDC_chkbox_show_debug_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>dlg_settings_chkbox_show_debug_info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="IDC_bSave">
     <property name="text">
      <string>dlg_set_btn_save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="IDC_bCancel">
     <property name="text">
      <string>dlg_set_btn_cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>IDC_bSave</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogSettings</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_bCancel</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogSettings</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_bCacheDir</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogSettings</receiver>
   <slot>select_cache_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_pbHelpCache</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogSettings</receiver>
   <slot>help_cache_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_bAuthCfgClear</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogSettings</receiver>
   <slot>authcfg_clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_bAuthCfgEdit</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogSettings</receiver>
   <slot>authcfg_edit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>cancel()</slot>
  <slot>select_cache_dir()</slot>
  <slot>test_ckan_url()</slot>
  <slot>pre_ckan_api()</slot>
  <slot>help_cache_dir()</slot>
  <slot>help_pre_urls()</slot>
  <slot>help_api_url()</slot>
  <slot>choose_pre_api()</slot>
  <slot>authcfg_edit()</slot>
  <slot>authcfg_clear()</slot>
 </slots>
</ui>
